dlp/snippets/deid/deid_test.go

```diff
@@ -570,3 +570,22 @@ func TestReidTextDataWithFPE(t *testing.T) {
 		t.Errorf("reidentifyFreeTextWithFPEUsingSurrogate got %q, want %q", got, want)
 	}
 }
+
+func TestDeIdentifyTableWithMultipleCryptoHash(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	var buf bytes.Buffer
+
+	if err := deIdentifyTableWithMultipleCryptoHash(&buf, tc.ProjectID, "your-transient-crypto-key-1", "your-transient-crypto-key-2"); err != nil {
+		t.Fatal(err)
+	}
+	got := buf.String()
+	if want := "Table after de-identification :"; !strings.Contains(got, want) {
+		t.Errorf("TestDeIdentifyTableWithMultipleCryptoHash got %q, want %q", got, want)
+	}
+	if want := `{string_value:\"my email is user1@example.org and phone is 858-555-0222\"}`; strings.Contains(got, want) {
```

Comment:
Please assert to confirm both info types have been de-identified.

Reply:
Updated.